src/app/admin/platform-config/platform-config.component.html

```diff
@@ -5,17 +5,17 @@ <h1 class="app-page-title">PLATFORM CONFIGURATION</h1>
         <ng-container *ngIf="(updateForm$ | async) as updateForm; else spinner">
             <form [formGroup]="updateForm" id="updateCoopForm">
                 <label for="name" class="app-page-label">Instance name</label>
-                <input class="form-control p-4" id="name" formControlName="name"
+                <input class="input-reg w-100" id="name" formControlName="name"
                        placeholder="Instance Name" type="text">
-
+    
```

Comment:
In this file, you have added several tabs instead of blank lines. Your editor should remove it automatically on save, I wonder why it didn't.